perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -16,6 +16,10 @@ sub bin_addresses_for_postcode {
 
     my $addresses = $self->whitespace->GetAddresses($postcode);
 
+    # Spec - Results can be returned as an array if more than one result is found.
+    # So presuming one address may not be an array
+    $addresses = [ $addresses ] unless ref $addresses eq 'ARRAY';
```

Comment:
I wonder if this might be better done in `perllib/Integrations/Whitespace.pm`, similar to the existing `GetStreets` method? That way any future callers of this method can get a consistent return value.